client/src/main/java/io/pravega/client/stream/impl/ReaderGroupStateManager.java

```diff
@@ -330,6 +337,21 @@ private void fetchUpdatesIfNeeded() {
             compactIfNeeded();
         }
     }
+
+    void updateConfigIfNeeded() {
+        sync.fetchUpdates();
```

Comment:
Shouldn't this be `fetchUpdatesIfNeeded()`?

Reply:
Changed to use `fetchUpdatesIfNeeded()`.